### PR TITLE
Cleaner and consistent design of front end footer

### PR DESCRIPTION
**Pull request by drummer83**
#### What? Why?

In the current footer design I found four things which are not perfectly designed:

1. Tiny font for important links below 'Keep in touch', 'Navigate' and 'Join us'
2. Misaligned rows between 'Keep in touch' and 'Navigate'
3. Different font sizes of text and link below 'Join us' 
4. Different font size of last paragraph in the legal section

footer old

#### What should we test?
- Re 1: Font size of links is slightly increased
- Re 2: Rows are well aligned on all relevant screen sizes (unless there is a line break)
- Re 3: Text and link below 'Join us' have the same font size
- Re 4: Text of last paragraph in the legal section has the same font size as the other legal paragraphs
- General: Responsive behaviour is still working as expected
- General: Design is good on all relevant screen sizes

#### Release notes
Cleaner and consistent design of front end footer

Changelog Category: User facing changes

### CODE REVIEW

File: app/assets/stylesheets/darkswarm/footer.scss [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 footer {
   .row {
-    p a {
-      font-size: 0.875rem;
+    p a {  /* was p a  */
```
Comment (jibees) [inline app/assets/stylesheets/darkswarm/footer.scss]: Is there any reason to keep history as comment? I think `git` can do the same no?
_Edit:_ Ok, I saw the next commits ;)

File: app/assets/stylesheets/darkswarm/footer.scss [hunk 2]
```diff
@@ -4,8 +4,8 @@
 
 footer {
   .row {
-    p a {  /* was p a  */
-      font-size: 1rem; /* was 0.875rem */
+    p a {
```
Comment (jibees) [inline app/assets/stylesheets/darkswarm/footer.scss:7]: I think this commit should be squashed with the previous one (to have a clean history). See https://github.com/openfoodfoundation/openfoodnetwork/wiki/Making-a-great-pull-request#keep-your-commit-history-clean-tidy-and-up-to-date for more.

File: app/assets/stylesheets/darkswarm/footer.scss [hunk 3]
```diff
@@ -130,7 +130,7 @@ footer {
     }
 
     .legal a {
-       font-size: 0.875rem;
+      font-size: 0.875rem;
```
Comment (jibees) [inline app/assets/stylesheets/darkswarm/footer.scss:133]: This commit should also be squashed with the previous one. 🙏

File: app/assets/stylesheets/darkswarm/footer.scss [hunk 4]
```diff
@@ -89,7 +89,7 @@ footer {
 
     @include panepadding;
 
-    .row a img {
+    div.row a img {
```
Comment (jibees) [inline app/assets/stylesheets/darkswarm/footer.scss]: This commit should be squashed with the previous one 🙏 .

**Review by jibees — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution @drummer83 ! 
I requested some changes, like squashing all the css modifications into one single commit.
Also I'm wondering what is this commit : #b1e7915 ? As far as I know, Il will delete it.

**Comment by sauloperez** [comment]
Nice @drummer83 ! thanks a lot for this contribution! could you please add a screenshot with the state after your changes? this will enable anyone to validate them, especially the product team.

**Comment by drummer83** [comment]
Hey, @sauloperez, @jibees,
I am so sorry for this mess. I intended to keep this pull request as "draft" only - to have it prepared but not finalized. I wasn't aware that you get informed about this already...
I will try to tidy everything up and once I am done we will come back to this!
Thank you for all your comments anyway! :-)

**Comment by drummer83** [comment]
To clean up this mess I will close this PR and create a new one. 
Sorry!

**Pull request closed without merging**